public/style.css

```diff
@@ -25,6 +25,11 @@ li {
   margin: auto;
 }
 
+#user .home-label {
+  background: linear-gradient(#000,  #00b0ff);
```

Comment:
I dont suppose this blue gradient is looking good. Doesn't the pure black look better?

Reply:
Pure black was fine but I thought that we should highlight the useful area by making a blue gradient.

Reply:
Okay, understood. In my opinion, doesn't look very aesthetically pleasing.

Reply:
With the current file, I can make it look like this with a small commit. I can raise a new Issue on this repository and make this change according to that.

![image](https://user-images.githubusercontent.com/20207952/94999717-f23b9d00-05d8-11eb-9189-14be5245f1e8.png)
However, if it doesn't fit the requirement, please let me know when you raise a new Issue specifying the required look for this. I'd like to work on that.

Reply:
@kots14 Let's go with 2. Since Lalit wanted to insert a blue shade. Can you make a PR for this?

Reply:
One more problem this PR creates is the git history. After merging this PR the git history is clumsy. I am gonna revert all changes done in this PR and will merge your next PR with the changes requested by @abhigyank and these changes. Make sure to that you have added all these changes in new PR. If you need the present state of repository for any reference then check [70-merged](https://github.com/KamandPrompt/baat-cheet/tree/70-merged) branch @kots14.

Reply:
@LalitNM Thanks for creating the reference branch. I'll raise a new PR with the changes mentioned by @abhigyank.

Reply:
Added new PR https://github.com/KamandPrompt/baat-cheet/pull/82 for https://github.com/KamandPrompt/baat-cheet/issues/66